utils/config.env

```diff
@@ -1,6 +1,28 @@
 #####################################################
-# Values in this file will be used to replace variables in script files
-# and Docker Compose files throughout the repo
+# The following values are provided by Confluent's
+# release automation process.  The values in these
+# names will be adopted in scripts and expressions
+# over time to construct other
+# values required by this repository
+#####################################################
+CONFLUENT=5.5.0-SNAPSHOT
+CONFLUENT_DOCKER_TAG=5.5.x-latest
+CONFLUENT_SHORT=5.5
+CONFLUENT_PREVIOUS=5.4.2
+CONFLUENT_RELEASE_TAG_OR_BRANCH=5.5.x
+CONFLUENT_MAJOR=5
+CONFLUENT_MINOR=5
+CONFLUENT_PATCH=0
+KAFKA=5.5.0-SNAPSHOT
```

Comment:
Are KAFKA and CE_KAFKA switched?

Reply:
I'm not certain... @andrewegel does it matter?  I pulled these from the companion spreadsheet.

Reply:
`KAFKA` is correct - `CE_KAFKA` would be `5.4.0-ce-SNAPSHOT` in this context.

Reply:
I just hand typed this in so it will be replaced by an appropriate value after we merge this and automation kicks in, is that correct?  Does it need to be a valid value before that happens?

Reply:
The value doesn't really matter, this is an idempotent process, if the script encounters any value, it'll just put in the right one.